src/Nordea/Components/Error.elm

```diff
@@ -0,0 +1,222 @@
+
+
+pageNotFound : Locale -> Error msg
+pageNotFound locale =
+    init { errorType = PageNotFound, locale = locale }
+
+
+init : ErrorConfig -> Error msg
+init config =
+    Error { locale = config.locale, errorType = config.errorType }
+
+
+view : List (Attribute msg) -> List (Html msg) -> Error msg -> Html msg
+view attributes children (Error config) =
+    let
+        toI18NString textToTranslate =
+            config.locale textToTranslate
+
+        errorDescription =
+            if config.errorType == PageNotFound then
+                texts.pageNotFound.description |> toI18NString
+
+            else
+                texts.internalServerError.description |> toI18NString
+
+        errorAction =
+            if config.errorType == InternalServerError then
+                texts.internalServerError.action |> toI18NString
+
+            else
+                ""
+    in
+    Html.div
+        [ Attributes.css
+            [ Css.backgroundColor Colors.white
+            , Css.width (pct 100)
+            , Css.height (vh 100)
+            , Css.padding (rem 1.5)
+            , Css.displayFlex
+            , Css.flexDirection Css.column
+            , Css.alignItems Css.center
+            ]
+        ]
+        [ Html.div [ Attributes.css [ Css.marginBottom (rem 1) ] ]
+            [ Text.headlineTwo
+                |> Text.withHtmlTag Html.h1
+                |> Text.view
+                    [ Attributes.css [ Css.textAlign Css.center ]
+                    ]
+                    [ Html.text (texts.heading |> toI18NString) ]
+            ]
+        , Html.div
+            [ Attributes.css
+                [ Css.marginBottom (rem 2)
+                , Css.maxWidth (rem 30)
+                , Css.textAlign Css.center
+                ]
+            ]
+            [ Text.bodyTextLight
+                |> Text.view []
+                    [ Html.text errorDescription
+                    ]
+            ]
+        , Html.div [] [ errorSvg ]
+        , showIf
+            (config.errorType
+                == InternalServerError
+            )
+            (viewActionForInternalServerError errorAction)
+        , Html.div
+            (Attributes.css [ Css.maxWidth (rem 30) ] :: attributes)
+            children
+        ]
+
+
+viewActionForInternalServerError : String -> Html msg
+viewActionForInternalServerError actionText =
+    Html.div
+        [ Attributes.css
+            [ Css.displayFlex
+            , Css.flexDirection Css.column
+            , Css.maxWidth (rem 30)
+            , Css.alignItems Css.center
+            ]
+        ]
+        [ Text.bodyTextLight
+            |> Text.view [ Attributes.css [ Css.paddingRight (rem 0.25), Css.textAlign Css.center ] ]
+                [ Html.text actionText
+                , FlatLink.default |> FlatLink.view [ href "mailto: kundeservice.nfe@nordea.com", Attributes.css [ Css.display Css.inlineBlock ] ] [ Html.text "kundeservice.nfe@nordea.com." ]
+                ]
+        ]
```

Comment:
```suggestion
viewActionForInternalServerError : String -> Html msg
viewActionForInternalServerError actionText =
    Text.bodyTextLight
            |> Text.view 
                [ Attributes.css 
                    [ Css.paddingRight (rem 0.25)
                    , Css.textAlign Css.center 
                    , Css.displayFlex
                    , Css.flexDirection Css.column
                    , Css.maxWidth (rem 30)
                    , Css.alignItems Css.center
                    ] 
                 ]
                [ Html.text actionText
                , FlatLink.default |> FlatLink.view [ href "mailto: kundeservice.nfe@nordea.com", Attributes.css [ Css.display Css.inlineBlock ] ] [ Html.text "kundeservice.nfe@nordea.com." ]
                ]
```

Reply:
The same comment her as the one further up. The text component clears the marginBottom because it has margin: 0

Reply:
But does the div have a margin in this case? 🙂